app/models/item.rb

```diff
@@ -19,14 +19,13 @@ def initialize(attrs = {})
 
     super
 
-    sign = SignModel.resolve.fetch_by_id_via_cache(sign_id)
-
+    sign = SignModel.resolve.find(sign_id)
     return if sign.nil?
 
     self.id         = sign.id          if id.nil?
     self.name       = sign.gloss_main  if name.nil?
     self.maori_name = sign.gloss_maori if maori_name.nil?
-    self.drawing    = sign.drawing     if drawing.nil?
```

Comment:
While the glosses can be overridden on the vocab sheet, the drawing cannot, so we don't need a condition on this